rfc002-procope-blog.md

```diff
@@ -0,0 +1,208 @@
+	* We want to encourage publisher rotation. This could mean one publisher steps down around the time of PSC transitions (around March) or six months afterward, allowing an interested editor to fill that role. This means part of being an editor is getting the specific experience needed to take on the role of publisher. (The first handoff may be bumpy, so we'll play this by ear.)
+	* First publishers would be: Ryan Macklin & Carrie Crowe.
+	* The GDP chair is also in this role, with respect to account access.
+* There are **editors**, who edit pieces to align with our community standards and style. We'll want two editors—and only two editors—involved in any one piece, to keep editors from burning out and to keep the signal/noise ratio down.
+	* Publishers are also editors, and it should go without saying that an author is never their own editor.
+* Anyone in the Procope group can be a **reviewer** when reviews are solicited. Pieces should have reviews solicited.
+* Anyone in the Procope group can submit a proposal for a piece they want to create. Such pieces would be one-off blog posts or videos, and when green lit will go through the process of drafting, reviews, edits, and finally entering the publication queue.
+	* A few authors may do a series, but that will be limited—partly because our publication queue can't maintain several series, and partly because we want to see what contributors provide to the group before promising to support a series.
+* Anyone in the Procope group can add to a topic wishlist.
+
+### Publication rights & ownership
+
+The creators of each piece will retain ownership of their pieces, and the Good Docs Project will effectively license the rights to publish that version of their piece in perpetuity.
+
+This is largely contrary to other open-source stances, but is important for three reasons:
+
+1. We want to allow contributors a space to incubate their ideas, which means in some cases a space to develop IP they want to work with later—such as in a talk or book down the road. And we want our contributors to feel good about having added to GDP's body of knowledge, rather than regret "giving up a good idea."
+2. There is an unfortunate history of privileged classes taking work created by marginalized people, and profiting off of that work without recognition of or compensation to those people. Because of the individual nature of each post and video, we should strive to not be a part of that history.
+3. Such media doesn't need to be replicated and altered in the way code bases are. Code bases and similar need legal declarations to protect those using them. The replication and alteration process is a personally interpreted one, and it doesn't necessitate a legal declaration.
+
+### Publication cadence
```

Comment:
thought (non-blocking): A potential downside of setting a cadence is that if there is the quality of material isn't good enough, or the reviewers run out of bandwidth, the forum degrades to low quality spam. I think it would be valuable to have a sentences or two talking to how you will keep the quality of the forum high and targeted.

Reply:
thought (non-blocking): We could reward people's work without a restrictive license by using our Open Collective funds to give $$ to the "best" blog post per quarter, as decided by the PSC. We could also set up a tip jar to be split among all the opted-in contributors, with a strong suggestion that salaried/mid-to-late-career TWs leave it for the junior/freelance/marginalized folks.

Reply:
That's a good point, Cameron. I used to run a (now-closed) decade-long blog where I got up to 8-10 posts a month, and most were good. Some were duds. I downshifted after around a year of that volume, to more like once a week. I think we should be able to downshift. We might not recognize yet when that downshift needs to happen, so I wouldn't want to proceduralize it.

Once there's a $$$ thing attached, sentiments shift a lot. It goes from a public good to a sort of lottery, and I'd worry about souring people who weren't awarded in a given time from feeling like their contributions were "lesser."